paasta_tools/chronos_tools.py

```diff
@@ -185,6 +186,13 @@ def __eq__(self, other):
             (self.branch_dict == other.branch_dict)
         )
 
+    def __normalize(self):
+        """When the config supports alternative syntaxes, convert them
+        to a normal form."""
+        parents = self.get_parents()
+        if parents is not None and not isinstance(parents, list):
+            self.config_dict['parents'] = [parents]
```

Comment:
> What if you moved this code to the get_parents getter?

I thought about that, but the [\_\_eq\_\_](https://github.com/Yelp/paasta/blob/c57563ea7a6c6ab1d80bc03c5bb169581e3e8fc9/paasta_tools/chronos_tools.py#L181) method does something like "`self.config_dict == other.config_dict`" when testing `ChronosJobConfig` objects for equality. If I don't "normalize" the config, alternative syntaxes would fail the `__eq__` test even when they express the same thing in different ways (`parents: 'blah'` vs. `parents: ['blah']`).

Happy to patch the getter, if I'm overthinking. :)